src/store/modules/job/actions.ts

```diff
@@ -525,7 +525,7 @@ const actions: ActionTree<JobState, RootState> = {
       'SERVICE_COUNT': '0',
       'SERVICE_TEMP_EXPR': job.jobStatus,
       'jobFields': {
-        'productStoreId': this.state.user.currentEComStore.productStoreId,
+        'productStoreId': job.status === "SERVICE_PENDING" ? job.productStoreId : this.state.user.currentEComStore.productStoreId,
```

Comment:
Add the check: https://github.com/hotwax/job-manager/pull/211/files#diff-1edc9153c69edc14cbca5528effde857efdaf6da5ac6048e4d6fc896b11ffcb4R540

Reply:
Yes sir. Understood why we need this check. Fixed it